src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1295,5 +1296,24 @@ public override void VisitStaticLocalInitializationSemaphore(IStaticLocalInitial
             Assert.NotNull(operation.Local);
             Assert.True(operation.Local.IsStatic);
         }
+
+        public override void VisitReDim(IReDimOperation operation)
+        {
+            Assert.Equal(OperationKind.ReDim, operation.Kind);
+            AssertEx.Equal(operation.Clauses, operation.Children);
+            var preserve = operation.Preserve;
+        }
+
+        public override void VisitReDimClause(IReDimClauseOperation operation)
+        {
+            Assert.Equal(OperationKind.ReDimClause, operation.Kind);
+            Assert.NotNull(operation.Operand);
```

Comment:
>Assert.NotNull(operation.Operand); [](start = 12, length = 34)

I believe this assert and similar assert in the loop below is redundant because elsewhere we assert that there are no nulls in Children, therefore the last assert in this function effectively checks these conditions too.